### DISCUSSION TITLE
Export audio volume is low

### DISCUSSION THREAD

**Discussion by HOUMOO** [Rainbow-Dreamer/musicpy#62] [General]
I am calling my own soundfont file, using the daw class to export audio, and the volume is very small, only 17db. What is the reason for this, or how can this be improved?

**Comment by Rainbow-Dreamer** [comment]
Hi, this is because musicpy's daw module is using fluidsynth to render soundfont file to audio, and the default settings of gain (global volume) of fluidsynth is set to the default value 0.2, you can change the value of gain to a larger value, for the `sf2_loader` instances in the `channel_instruments` attribute of a `daw` instance, call the function `change_settings`, for example,
```python
current_daw.channel_instruments[0].change_setting('gain', 1)
```
and then export audio again.
